--- a/net/minecraft/client/renderer/block/ModelBlockRenderer.java
+++ b/net/minecraft/client/renderer/block/ModelBlockRenderer.java
@@ -323,6 +_,11 @@
         }
     }
 
+    @Deprecated
+    public void renderModel(PoseStack.Pose p_111068_, VertexConsumer p_111069_, @Nullable BlockState p_111070_, BakedModel p_111071_, float p_111072_, float p_111073_, float p_111074_, int p_111075_, int p_111076_) {
+        renderModel(p_111068_, p_111069_, p_111070_, p_111071_, p_111072_, p_111073_, p_111074_, p_111075_, p_111076_, net.neoforged.neoforge.client.model.data.ModelData.EMPTY, null);
+    }
+
     public void renderModel(
         PoseStack.Pose p_111068_,
         VertexConsumer p_111069_,
@@ -332,7 +_,9 @@
         float p_111073_,
         float p_111074_,
         int p_111075_,
-        int p_111076_
+        int p_111076_,
+        net.neoforged.neoforge.client.model.data.ModelData modelData,
+        net.minecraft.client.renderer.RenderType renderType
     ) {
         RandomSource randomsource = RandomSource.create();
         long i = 42L;
