--- a/net/minecraft/world/level/block/entity/BlockEntity.java
+++ b/net/minecraft/world/level/block/entity/BlockEntity.java
@@ -6,6 +_,7 @@
 import java.util.Set;
 import javax.annotation.Nullable;
 import net.minecraft.CrashReportCategory;
+import net.minecraft.client.player.LocalPlayer;
 import net.minecraft.core.BlockPos;
 import net.minecraft.core.HolderLookup;
 import net.minecraft.core.component.DataComponentMap;
@@ -295,6 +_,10 @@
             LOGGER.warn("Failed to parse custom name from string '{}', discarding", p_342033_, exception);
             return null;
         }
+    }
+
+    public boolean hasCustomOutlineRendering(LocalPlayer player) {
+        return false;
     }
 
     static class ComponentHelper {
