--- a/net/minecraft/client/renderer/LevelRenderer.java
+++ b/net/minecraft/client/renderer/LevelRenderer.java
@@ -3540,6 +_,10 @@
         return this.cloudsTarget;
     }
 
+    public Frustum getFrustum() { return this.capturedFrustum != null ? this.capturedFrustum : this.cullingFrustum; }
+
+    public void iterateVisibleBlockEntities(java.util.function.Consumer<BlockEntity> consumer) {}
+
     private void shootParticles(int p_307280_, BlockPos p_307603_, RandomSource p_307547_, SimpleParticleType p_307590_) {
         Direction direction = Direction.from3DDataValue(p_307280_);
         int i = direction.getStepX();
