--- a/net/minecraft/client/Minecraft.java
+++ b/net/minecraft/client/Minecraft.java
@@ -416,7 +_,7 @@
         RepositorySource repositorysource = new FolderRepositorySource(
             this.resourcePackDirectory, PackType.CLIENT_RESOURCES, PackSource.DEFAULT, this.directoryValidator
         );
-        this.resourcePackRepository = new PackRepository(clientpacksource, this.downloadedPackSource.createRepositorySource(), repositorysource);
+        this.resourcePackRepository = new PackRepository(clientpacksource, this.downloadedPackSource.createRepositorySource(), repositorysource, new org.embeddedt.phi.resources.PhiResourcePackRepository()); // Phi: Load our resources
         this.vanillaPackResources = clientpacksource.getVanillaPack();
         this.proxy = p_91084_.user.proxy;
         this.authenticationService = new YggdrasilAuthenticationService(this.proxy);
