--- a/net/minecraft/client/resources/model/SimpleBakedModel.java
+++ b/net/minecraft/client/resources/model/SimpleBakedModel.java
@@ -26,6 +_,7 @@
     protected final TextureAtlasSprite particleIcon;
     protected final ItemTransforms transforms;
     protected final ItemOverrides overrides;
+    protected final net.neoforged.neoforge.client.ChunkRenderTypeSet blockRenderTypes = null;
 
     public SimpleBakedModel(
         List<BakedQuad> p_119489_,
