--- a/net/minecraft/world/level/Level.java
+++ b/net/minecraft/world/level/Level.java
@@ -1100,6 +_,10 @@
 
     public abstract PotionBrewing potionBrewing();
 
+    public net.neoforged.neoforge.client.model.data.ModelDataManager getModelDataManager() {
+        return net.neoforged.neoforge.client.model.data.ModelDataManager.INSTANCE;
+    }
+
     public static enum ExplosionInteraction implements StringRepresentable {
         NONE("none"),
         BLOCK("block"),
